modules/sdk-coin-polygon/test/unit/transactionBuilder/walletInitialization.ts

```diff
@@ -141,5 +141,41 @@ describe('Polygon wallet initialization', function () {
       txBuilder.counter(1);
       await txBuilder.build().should.be.rejectedWith('Invalid transaction: missing contract call data field');
     });
+
+    it('should build and sign using txHex', async () => {
```

Comment:
Why are we adding the test case in polygon when we made a change in eth module? can't we write the tests there?

Reply:
sure anyway polygon uses eth transaction builder
added in eth tests